uint32 IsOrtho
float32 VisionAngle
float32 nearClippingPlane
float32 FarClippingPlane
---
bool camera_parameters
